src/components/app.tsx

```diff
@@ -409,7 +425,10 @@ export class App extends React.PureComponent<IProps, IState> {
             { showOfflineNav && <OfflineNav onOfflineActivities={this.handleShowOfflineActivities} /> }
             { this.renderContent() }
             { this.state.showThemeButtons && <ThemeButtons/>}
-            <div className="version-info" data-cy="version-info">{(window as any).__appVersionInfo || "(No Version Info)"}</div>
+            <div className="version-info" data-cy="version-info">
+              Application: {appVersionInfo || "No Version Info"}
+              {serviceWorkerVersionInfo && ` | Service Worker: ${serviceWorkerVersionInfo} ${mismatchedVersions ? "❌" : ""}`}
```

Comment:
I added this red X so we can ask "Is there a red X at the bottom of the page?" instead of asking them to compare the version numbers themselves.  I did have a ✓ if the versions matched but it looked out of place so I removed it.